Sources/HipMobileUI/App.xaml

```diff
@@ -36,13 +38,135 @@
             <Color x:Key="LightGreenBarColor">#A9D839</Color>
             <Color x:Key="YellowBarColor">#FDDB37</Color>
             <Color x:Key="OrangeBarColor">#FEB533</Color>
-            <Color x:Key="RedBarColor">#FF8E59</Color>       
+            <Color x:Key="RedBarColor">#F22F0C</Color>       
 
             <Style TargetType="Label">
                 <Setter Property="FontSize" Value="12" />
                 <Setter Property="TextColor" Value="Black" />
             </Style>
 
+            <Style x:Key="Ssp-Black" TargetType="Label">
```

Comment:
This is just a minor remark but could you order the fonts based on how they are listed in the Corporate Design Manual? It would make finding the key for a specific font easier.